lib/sentry/integrations/check_in_id_mappings.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Sentry.Integrations.CheckInIDMappings do
+  @moduledoc false
+
+  use GenServer
+
+  @table :cron
+
+  @spec start_link(keyword()) :: GenServer.on_start()
+  def start_link([] = _opts) do
+    GenServer.start_link(__MODULE__, nil, name: __MODULE__)
+  end
+
+  @spec lookup(String.t()) :: {:ok, <<_::256>>}
+  def lookup(key) do
```

Comment:
Nit, but can we call this something like `lookup_or_insert_new` to make it clear that this'll generate a new UUID if needed?

Also, the return value can just be `UUID.t()` here, no need for the `{:ok, _}` tuple either.